sdk/go/common/workspace/paths.go

```diff
@@ -242,28 +241,13 @@ func GetCachedVersionFilePath() (string, error) {
 
 // GetPulumiHomeDir returns the path of the '.pulumi' folder where Pulumi puts its artifacts.
 func GetPulumiHomeDir() (string, error) {
-	// Allow the folder we use to be overridden by an environment variable
-	dir := os.Getenv(PulumiHomeEnvVar)
```

Comment:
can we remove 	`PulumiHomeEnvVar = "PULUMI_HOME"` from the consts above as well?

Reply:
I addressed this in https://github.com/pulumi/pulumi/pull/13747/commits/1a7b930a100b1daba264259e75d6d3bec9839317. Quoting from the commit message:
> I am hesitant to remove the variable, since sdk/go/common/workspace is public and
`PulumiHomeEnvVar` is exported. I am comfortable moving it from a `const` to a `var`.